src/EnergyPlus/BranchInputManager.cc

```diff
@@ -2460,8 +2463,8 @@ namespace BranchInputManager {
             FoundBranchName = "";
             if (present(CompType) && present(CompName)) {
                 for (CpN = 1; CpN <= state.dataBranchInputManager->Branch(BrN).NumOfComponents; ++CpN) {
-                    if (!UtilityRoutines::SameString(CompType, state.dataBranchInputManager->Branch(BrN).Component(CpN).CType) ||
-                        !UtilityRoutines::SameString(CompName, state.dataBranchInputManager->Branch(BrN).Component(CpN).Name))
+                    if (!UtilityRoutines::SameString(CompType(), state.dataBranchInputManager->Branch(BrN).Component(CpN).CType) ||
+                        !UtilityRoutines::SameString(CompName(), state.dataBranchInputManager->Branch(BrN).Component(CpN).Name))
```

Comment:
Hmm, this one is catching me off guard a bit.  What is this parentheses operator doing?

Reply:
`CompType` is an `Objexx::Optional<std::string>` and we need a `string_view`. There is no implicit conversion for this, so I'm calling `()` to explicitly get a `const std::string` which can then implicitly be converted to a `std::string_view`